### PR TITLE
refactor: update option page theme

### PR DESCRIPTION
**Pull request by frank-zsy**
## Brief Information

This pull request is in the type of ([more info](https://github.com/angular/angular/blob/main/CONTRIBUTING.md#type) about types):

- [ ] build
- [ ] ci
- [ ] docs
- [ ] feat
- [ ] fix
- [ ] perf
- [x] refactor
- [ ] test

Related issues ([all available keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)):

## Details

I am really fond of the VSCode default dark theme configs. So I implement a new theme for the option page which follows VSCode dark theme styles.

It looks like this now:

image

Don't know if you like this or not, just fire a PR first.

And since GitHub and Gitee token procedure will be updated, I did not correct the input component position right now.

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/hypertrons/hypertrons-crx/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx)

### CODE REVIEW

File: src/components/TooltipTrigger/index.tsx [hunk 1]
```diff
@@ -13,8 +13,8 @@ interface ITooltipTriggerProps {
 const TooltipTrigger: React.FC<ITooltipTriggerProps> = ({
   size = 20,
   iconColor = '#FFFFFF',
-  tooltipBackground = '#FFFFFF',
-  tooltipFontColor = '#242A2E',
+  tooltipBackground = '#333333',
+  tooltipFontColor = '#FFFFFF',
```
Comment (wangyantong2000) [inline src/components/TooltipTrigger/index.tsx:17]: Directly modifying will cause all colors using TooltipTrigger to change.
image

Comment (wangyantong2000) [reply src/components/TooltipTrigger/index.tsx:17]: `<TooltipTrigger tooltipFontColor={tooltipFontColor} tooltipBackground={tooltipBackground} content={t('options_locale_toolTip')} />`

Perhaps it is possible to set the values of its related colors during specific use, like this.

**Comment by wangyantong2000** [comment]
It looks beautiful.

File: src/components/TooltipTrigger/index.tsx (hunk 1 above)
Comment (frank-zsy) [reply src/components/TooltipTrigger/index.tsx:17]: Yes, pass the tooltip color as props may not be quite a good practice I think, can we extract the styles into the css, so option page can override the css style to customize its color?

**Comment by Xsy41** [comment]
Feel that the content and the background have a little degree of differentiation, will be more beautiful.

image

**Comment by frank-zsy** [comment]
> Feel that the content and the background have a little degree of differentiation, will be more beautiful.
> 

I am not quite sure because VSCode setting does not have a background for main container area, this makes it consistent for different screen resolution.

**Comment by wangyantong2000** [comment]
OK

**Comment by wangyantong2000** [comment]
There is indeed no distinction made in VSCode. Since it is in VSCode style, let's focus on the original VSCode style first. Continuously make adjustments in subsequent usage feedback. I will first resolve the conflict of this PR and then merge it.

**Comment by wangyantong2000** [comment]
image

Merge conflicts.
I'm not sure if it should be a circular button or a right angled button.

**Comment by wangyantong2000** [comment]
And after this PR is merged, a new version can be released. @Xsy41

**Comment by frank-zsy** [comment]
@wangyantong2000 I think right-angled button is good.
image

**Comment by frank-zsy** [comment]
@Xsy41 And https://hypercrx.cn is also ready now for the repo https://github.com/hypertrons/hypertrons-crx-website , you can also start to add the content of user docs on the website.

**Comment by wangyantong2000** [comment]
> @wangyantong2000 I think right-angled button is good. image

This doesn't look like a perfect right angle, it's like a circular button with a very small radius

**Comment by wangyantong2000** [comment]
截屏2024-11-18 22 10 30

**Comment by frank-zsy** [comment]
I am not quite sure, you can decide which style to use since the css is generated by GPT-4~

**Comment by Xsy41** [comment]
> After the refactoring work on the options page is merged, you can update and modify the comments on the token related part of the FastPR process in [911](https://github.com/hypertrons/hypertrons-crx/issues/911) again~ And after this PR is merged, a new version can be released. And https://hypercrx.cn/ is also ready now for the repo https://github.com/hypertrons/hypertrons-crx-website , you can also start to add the content of user docs on the website.

OK, I’ll update the token-related comments for [#911](https://github.com/hypertrons/hypertrons-crx/issues/911) after the refactoring work is merged and start adding user documentation to https://hypercrx.cn/ for the website repo.

**Pull request merged**